src/mero_wrk_tcp_txt.erl

```diff
@@ -109,6 +109,19 @@ transaction(Client, delete, [Key,  TimeLimit]) ->
             {error, Reason}
     end;
 
+transaction(Client, mdelete, [Keys,  TimeLimit]) ->
+    lists:foreach(fun(Key) ->
+                          case send_receive(Client, {?MEMCACHE_DELETE, {Key}}, TimeLimit) of
+                              {ok, deleted} ->
+                                  {Client, ok};
+                              {error, not_found} ->
+                                  {Client, {error, not_found}};
```

Comment:
Ah! You're right. I thought "eh, fire and forget just keep trying" but, yeah, it would be better to bail out faster.